pkgs/servers/gonic/default.nix

```diff
@@ -55,7 +55,7 @@ buildGoModule rec {
     description = "Music streaming server / subsonic server API implementation";
     license = lib.licenses.gpl3Plus;
     maintainers = with lib.maintainers; [ autrimpo ];
-    platforms = lib.platforms.linux;
+    platforms = lib.platforms.unix;
```

Comment:
```suggestion
```
I don't think we need to limit the platforms to unix.
If not set, the default value is used.
https://github.com/NixOS/nixpkgs/blob/d8e31f66170f106c559e7021f1dfcc71b58fab94/pkgs/build-support/go/module.nix#L310-L313

Reply:
Isn't being explicit about the platforms we know work well/are supported better than relying on an implicit default value?

Reply:
I agree that the defaults here make sense, because it's a go program, and is likely to work anywhere that go works. So I've made the suggested change, thanks.